### DISCUSSION TITLE
[API] Snapshots interval

### DISCUSSION THREAD

**Discussion by wrkilu** [grafana/grafana#51352] [Dashboards, Panels & Visualizations]
Hi,
In current API there isn't option for seting interval for creating snapshots in Synthetics Monitoring in tcp checks. And I think this is very basic option for snapshots.
Thank you, regards.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
